### DISCUSSION TITLE
SSO ? (single sign on)

### DISCUSSION THREAD

**Discussion by hadhub** [dbarzin/mercator#659] [Ideas]
Bonjour Didier Barzin,

Le SSO est prévu d'être intégré au sein du projet ?

En vous souhaitant une bonne journée.

Hadrien Chauder

**Comment by dbarzin** [comment]
L'application est écrite avec le Framework Laravel. Il existe des packages pour intégrer un SSO ce framework.
N'ayant que peu d'expérience en la matière, cela se verrait plus facilement via une contribution au projet :smile:.

**Reply by hadhub** [reply]
Je n'ai pas énormément d'expérience également mais je vais voir ça (peut être avec un collègue) 😄
Merci pour votre réponse

**Reply by ChrisBear2980** [reply]
J'ai trouvé la procédure suivante : [https://stackoverflow.com/questions/54289010/azure-active-directory-sso-with-laravel](https://stackoverflow.com/questions/54289010/azure-active-directory-sso-with-laravel)
A tester...

**Reply by dbarzin** [reply]
Attention, cela date d'il y a 6 ans. A tester.

**Reply by dbarzin** [reply]
Mercator utilise KeyCloack comme solution de SSO : https://github.com/dbarzin/mercator/blob/master/INSTALL.fr.md#configuration-de-keycloak-optionel
